lib/vdc/DCBOV.cpp

```diff
@@ -286,7 +286,8 @@ template<class T> void DCBOV::_generateCoordinates(int dim, const vector<size_t>
     std::array<double, 3> origin = _bovCollection->GetBrickOrigin();
     std::array<double, 3> brickSize = _bovCollection->GetBrickSize();
 
-    double increment = brickSize[dim] / (dataSize[dim] - 1);
+    double denom = (dataSize[dim] - 1);
+    double increment = denom == 0 ? 0 : brickSize[dim] / denom;
```

Comment:
Scott, do you remember we were talking about making your intentions clear to the compiler (and to your code reviewers) by explicitly expressing how you would like type conversions to happen? 🤔🤔🤔

Reply:
Missed that one.  Fixed.

Are you using a static analyzer to find these problems?  When I run clang-tidy on this file it reports no warnings.

Reply:
Just spotted it on this github page. clang-tidy sounds like something could be helpful to us :)